src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -76,6 +77,30 @@ public function iCancelThisOrder(OrderInterface $order): void
         );
     }
 
+    /**
+     * @When /^I mark (this order) as paid$/
+     */
+    public function iMarkThisOrderAsAPaid(OrderInterface $order): void
+    {
+        $this->client->customEndPoint(
+            sprintf('/payments/%s/complete', (string) $order->getLastPayment()->getId()),
+            'PATCH',
+            $this->sharedStorage->get('token')
+        );
+    }
+
+    /**
+     * @When /^I ship (this order)$/
+     */
+    public function iShipThisOrder(OrderInterface $order): void
+    {
+        $this->client->customEndPoint(
+            sprintf('/shipments/%s/ship', (string) $order->getShipments()->first()->getId()),
```

Comment:
That's why we need some possibility to gather links and available actions :(

Reply:
Another question is, do we want to expose these actions also as sub resources of orders (what we are doing in admin panel). Perhaps not?

Reply:
Perhaps... I don't know 🤷‍♂. On the one hand, It seems reasonable to access shipments via orders. You check out the orders list (`orders/`), then the specific order (`orders/000111`), you want to see its shipments (`orders/000111/shipments`), especially one of them (`orders/000111/shipments/12`), and to ship it (`orders/000111/shipments/12/dancer`) 💃 But it seems like a long way and probably an improvement to the base as it is right now 🖖